packages/hw-app-ada/src/Ada.js

```diff
@@ -57,8 +57,7 @@ export default class Ada {
     this.transport = transport;
     this.methods = [
       "isConnected",
```

Comment:
let's rename this to `getVersion`. (Please cross-check with BTC what naming do they use)

Reply:
From my fast look at the BTC app, looks like they dont have app version call.